pco/src/delta.rs

```diff
@@ -125,37 +125,34 @@ fn lookback_hash_lookup(
   }
 }
 
+#[inline(never)]
 fn lookback_compute_goodness<L: Latent>(
```

Comment:
We should change the name of this function now

Reply:
Done